dqsegdb2/http.py

```diff
@@ -18,17 +18,30 @@
 """HTTP interactions with DQSEGDB
 """
 
+import os
 import json
+import scitokens
+from scitokens.utils.errors import InvalidTokenFormat
+from jwt.exceptions import InvalidAudienceError
```

Comment:
Can you please separate these third-part imports from the standard library imports?